### DISCUSSION TITLE
One user(admin) having incostitent results submitting forms

### DISCUSSION THREAD

**Discussion by AbbeyDesign** [solspace/craft-freeform#411] [Questions & Answers]
I have a form that submits, creates an entry, and notifies specific users.

It works. Except for one admin account. They can sometimes submit, but more often than not, they get an error.

But it works from my account, if I log in as them, or if I log in as another admin account. All with the same permissions. I also have vpn'd to be in the same timezone as them.

They keep getting a "There was an error submitting this form," but it never lists the error, and there is nothing in the error log.

I know this is vague, but without any indication of what is wrong, I am looking for some direction of where else to look in the logs or if there is a setting that I can change.

- Craft Version: 3.7.48
- Freeform Version: 3.13.14
- Freeform Edition: Pro

**Comment by kjmartens** [comment]
Hi @AbbeyDesign,

This one will be hard to troubleshoot, especially if you can't duplicate it yourself, and it's only happening to a single admin account. I would check with them to see if you can identify any patterns in their use of the site, etc.

One possible scenario could be that they leave the browser open all day and then attempt to use a stale form or something. Craft's CSRF token will expire after 2 or 3 hours I believe, so this could be a plausible scenario. 🙂
